app/views/my_modules/_module_header.html.erb

```diff
@@ -62,7 +62,7 @@
     </div>
     <div class="well well-sm">
       <span class="hidden-xs hidden-sm"><%=t "my_modules.module_header.tags" %></span>
-      <% if can_edit_module(@my_module) then %>
+      <% if can_manage_experiment?(@my_module.experiment) then %>
```

Comment:
remote  "then" is useless here

Reply:
I agree, but it's also used elsewhere. This should be left for refactoring.